deps/llvm-3.7.1.patch

```diff
@@ -0,0 +1,550 @@
+diff --git a/include/llvm/ExecutionEngine/SectionMemoryManager.h b/include/llvm/ExecutionEngine/SectionMemoryManager.h
+index 0b0dcb0..7bb96eb 100644
+--- a/include/llvm/ExecutionEngine/SectionMemoryManager.h
++++ b/include/llvm/ExecutionEngine/SectionMemoryManager.h
```

Comment:
i'm not convinced that we want to maintain this as a patch (instead of temporarily using a private copy of this class)